containers/ecr-viewer/src/app/admin/user/create/page.tsx

```diff
@@ -0,0 +1,52 @@
+import { revalidatePath } from "next/cache";
+import Link from "next/link";
+
+import { UserForm } from "@/app/admin/user/UserForm";
+import { ArrowBack } from "@/app/components/Icon";
+import { listProgramAreas } from "@/app/services/programAreaService";
+import {
+  createUserAction,
+  notFoundUnlessAdmin,
+  updateUserProgramAreasAction,
+} from "@/app/services/userService";
+
+/**
+ * @returns Page to add a new user
+ */
+const CreateUserPage = async () => {
+  await notFoundUnlessAdmin();
+
+  const programs = await listProgramAreas();
+
+  return (
+    <main className="main-container">
+      <div className="content-container margin-top-10">
+        <Link
+          href="/admin/user"
+          className="action-text margin-bottom-3 display-inline-flex flex-align-center"
+        >
+          <ArrowBack aria-hidden={true} className="square-3" />
+          Back to User Management
```

Comment:
```suggestion
          Back to user management
```

Reply:
I also changed it in the latest PR to say `Back to program area management` as its much easier as then we just have the item type and we madlib it wherever we need...

Reply:
either way, @angelathe this is getting wrapped into the form page content component, so I don't think you need to sweat this detail in this PR since it's gonna get eaten by the other one sooner or later